style.css

```diff
@@ -0,0 +1,312 @@
+}
+
+#searchbox {
+  width: 100%;
+  background-color: transparent;
+  border-radius: 10px;
+}
+
+.content-wrapper {
+  max-width: 1440px;
+  min-width: 800px;
+  margin: 0 auto;
+}
+
+.form-container {
+  border-radius: 10px;
+}
+
+.grid-wrapper {
+  max-width: 1440px;
+  min-width: 800px;
+  margin: 0 auto;
+}
+
+.hero {
+  background: url(https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
+  background-position: 50% 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-align: center;
+  height: 70vh;
+  display: grid;
+  align-content: space-evenly;
+  object-fit: cover;
+}
+
+.grid-header {
+  position: relative;
+  min-width: 800px;
+}
+
+.grid-footer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "info1 info2 info3 info4";
+  grid-gap: 2px;
+  justify-content: center;
+  height: 21.1vh;
+  max-width: 1440px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.hero-header {
+  padding: 50px 0;
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.main-header {
+  margin-bottom: -9px;
+}
+
+.head-img1 {
+  display: block;
+  height: 104px;
+  width: 100%;
+}
+
+.secondary-header {
+  margin-bottom: 27px;
+}
+
+.head-img2 {
+  display: block;
+  height: 82px;
+  width: 100%;
+}
+
+.watch-now {
+  height: 31px;
+  display: block;
+  overflow: hidden;
+}
+
+.watch-link {
+  margin: -32px 18px 25px;
+  display: inline-block;
+  height: 29px;
+}
+
+.head-img3 {
+  display: inline-block;
+  height: 64px;
+  width: 100%;
+}
+
+.info-shaper {
+  height: 21.1vh;
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+}
+
+.info-image {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.info-shaper .info-image img {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.info1 {
+  grid-area: info1;
+}
+
+.info2 {
+  grid-area: info2;
+}
+
+.info3 {
+  grid-area: info3;
+}
+
+.info4 {
+  grid-area: info4;
+}
+
+.footer-wrapper {
+  width: 100%;
+  height: 90px;
+  font-size: 0.8334em;
+  line-height: 1.2;
+  text-align: center;
+}
+
+.footer-container {
+  margin: 0 auto;
+  max-width: 80%;
+  min-width: 800px;
+  padding: 1em 0 2.6em;
+  color: #6e6e6e;
+  font-size: 0.6334em;
+  line-height: 1.2;
+  text-align: center;
+}
+
+.left {
+  padding-top: 1.5em;
+  text-align: left;
+  float: left;
+  width: 49%;
+}
+
+.footer-links {
+  width: 51%;
+  float: right;
+  list-style: none;
+  margin-top: 0.5em;
+}
+
+.footer-container .footer-links > li {
+  display: inline-block;
+  margin-left: 0.6em;
+  padding-left: 0.6em;
+  border-left: 1px solid #ccc;
+}
+
+.copyright-container {
+  text-align: left;
+  display: inline-block;
+  width: 100%;
+  clear: both;
+}
+
+.copyright {
+  display: inline-block;
+  padding-top: 1.5em;
+}
+
+.rights {
+  list-style: none;
+  display: inline-block;
+}
+
+.copyright-container .rights > li {
+  display: inline;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please keep `.rights` class and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_